cc3d/__init__.py

```diff
@@ -96,6 +96,9 @@ def get_formatted_version_info():
     pyqt_library_bin_path = python_exe_dir.joinpath('Library', 'bin')
     path_env_list = path_env.split(';')
 
+    # needed for maboss
+    mingw_bin_path = python_exe_dir.joinpath('Library', 'mingw-w64', 'bin')
```

Comment:
I was puzzled by this one too. It does not hurt to leave it for now but in the future we may want to think how we distribute CC3D. On my system this fix was needed because, while I was using conda-based python, the cc3d run scripts were executed under generic shell where nothing was assumed where the python distribution was coming from. I think in general we should not rely on assumiung that everyone will be using conda  . Another alternative is to run conda-activation scripts from within our run scripts. @tjsego  what do you think ?